server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -73,28 +72,43 @@ async fn requisition_transfer() {
     )
     .await;
 
-    let test = || async move {
+    let test_input = (
+        service_provider,
+        request_store,
+        response_store,
+        item1,
+        item2,
+    );
+
+    let number_of_instance = 6;
```

Comment:
:nit
should match the parameter name?
```suggestion
    let number_of_instances = 6;
```

Reply:
🙏, i changed globally: 7f22e618d631515aae33b8b47ed3f5941dfd4fec (I think my formatter was dead at some point, it's showing big diff, but that's due to extra spaces)